cli/src/main/resources/help/embed-host-controller.txt

```diff
@@ -0,0 +1,56 @@
+SYNOPSIS
+
+   embed-host-controller [-c=config_file || --domain-config=config=domain_config_file]
```

Comment:
Should be

... || --domain-config=domain_config_file

I think the text you have was a carry-forward of a typo I made in embed-server.txt

... || --server=config=config_file

The first '=' there should be a '-'. Can you fix that in this PR as well? Thanks.